mlflow/server/js/src/common/utils/StringUtils.js

```diff
@@ -13,3 +13,17 @@ export const capitalizeFirstChar = (str) => {
   }
   return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
 };
+
+export const middleTruncateKey = (keyName, maxLen) => {
+  if (keyName.length > maxLen) {
+    const firstPartLen = Math.floor((maxLen - 3) / 2);
+    const lastPartLen = maxLen - 3 - firstPartLen;
+    return (
+      keyName.substring(0, firstPartLen) +
+      '...' +
+      keyName.substring(keyName.length - lastPartLen, keyName.length)
+    );
+  } else {
+    return keyName;
+  }
+};
```

Comment:
```suggestion
export const middleTruncate = (s, maxLen) => {
  if (keyName.length > maxLen) {
    const firstPartLen = Math.floor((maxLen - 3) / 2);
    const lastPartLen = maxLen - 3 - firstPartLen;
    return (
      keyName.substring(0, firstPartLen) +
      '...' +
      keyName.substring(keyName.length - lastPartLen, keyName.length)
    );
  } else {
    return keyName;
  }
};
```

Can we use a more generic name here because this function can be applied any string?